### DISCUSSION TITLE
Liabilities - Difference between owe and owed

### DISCUSSION THREAD

**Discussion by viharm** [firefly-iii/firefly-iii#5926] [Firefly III]
Hi, I have borrowed from a friend. This was addressed by creating a liability account with the liability type `I owe this debt to somebody else`.

Now I've loaned him some money which is more than what I borrowed. So I've simply transferred money from one of my asset accounts into this liability account.

Since the liability type has changed from `I owe this debt to somebody else` to `I am owed this debt` do I need to change this in the account? Is there a mechanism that influences how this account works? Or should I create a separate account to handle this new transaction (even if it's the same friend)?

**Comment by JC5** [comment]
Either works, but I would create a new one myself.

**Reply by viharm** [reply]
Ok, thanks. So there aren’t any specific mechanics or math related to the liability type, right? It can be treated like any account?

**Reply by JC5** [reply]
It can be yes, there is some math related to the amount due but not that much.
